client/src/components/Reviews/components/ReviewList.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
+import ReviewList from 'Reviews/components/ReviewList.jsx';
+
+describe('test', () => {
```

Comment:
The describe text should specify the component being tested, i.e. `'ReviewList'`. That way when the test suit runs each block of tests will be organized by the component name. Otherwise we'll have a bunch of tests blocks that are just titled 'test', which won't be very helpful when we're trying to see where the failing test is.